modules/learn/pages/data/validity.adoc

```diff
@@ -0,0 +1,70 @@
+
+Couchbase Server provides xref:learn:clusters-and-availability/intra-cluster-replication.adoc[Intra-Cluster Replication]; which ensures data-availability with up to three replicas, deployed across the cluster.
+The _Database Change Protocol_ (DCP) keeps active and replica data-elements both consistent and up-to-date with _eventual consistency_.
+
+Additionally, Couchbase Server provides xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication (XDCR)]; which uses eventual consistency to replicate data-modifications across different clusters, potentially located in different data centers.
+
+The Couchbase xref:learn:services-and-indexes/services/index-service.adoc[Index Service] provides high-performance data-access; and as such, maintains fast-access copies of specified data.
+Such copies are updated with eventual consistency, across the cluster.
+A xref:n1ql:n1ql-rest-api/index.adoc#table_xmr_grl_lt[Read Your Own Writes] mechanism is available, to allow queries to read modified data from an index at the earliest point at which consistency is achieved.
+
+When data-elements must be updated consistently across multiple documents, xref:learn:data/transactions.adoc[Distributed ACID Transactions] can be used.
+
+[#isolation-definition-and-implementation]
+=== Isolation
+
+_Isolation_ ensures that no transaction's data-modifications are visible to any other transaction prior to commitment: a _transaction_ being an operation that performs an atomic update, involving multiple discrete data-elements.
+
+Couchbase xref:learn:data/transactions.adoc[Distributed ACID Transactions] provide the level of isolation referred to as https://jepsen.io/consistency/models/read-committed[Read Committed].
+This duly ensures that the uncommitted data of a transaction cannot be seen either by another transaction or by a simultaneous non-transactional operation.
+It also ensures that if two transactions try to update the same document, this is detected; and only one transaction is allowed to proceed.
+
+[#durability-definition-and-implementation]
+=== Durability
+
+_Durability_ guarantees that data written into a database is fully conserved; and is not lost even in the event of significant system-failure.
+
+Couchbase Server implements durability by means of _durable writes_ (sometimes referred to as _synchronous writes_).
+These may be appropriate when saving data whose loss could have a considerable, negative impact.
+For example, data corresponding to a financial transaction.
+A durable write must be completed on a majority of those cluster-nodes that bear the active and replica copies of the data, before the application is informed that the data has been committed.
+A successful commit may be considered to have occurred when the written data is either in the memory or on the disk of each of the nodes that constitute the majority.
```

Comment:
Maybe avoid the term commit here, given this is talking about single-docs and commit is an overloaded term for transactions.

Reply:
Yes. This is the language we've derived over time for the Durability page. I just copied it. I can probably just omit the last sentence for this summary.